docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -55,37 +55,61 @@ HOST_MOUNT_DIR=/data/dockerMount
 [[ -h /data/srv/wmagent ]] && rm -f /data/srv/wmagent
 ln -s $HOST_MOUNT_DIR/srv/wmagent /data/srv/wmagent
 
-
+# create the passwd and group mount point dynamically at runtime
+passwdEntry=$(getent passwd $wmaUser | awk -F : -v wmaHome="/home/$wmaUser" '{print $1 ":" $2 ":" $3 ":" $4 ":" $5 ":" wmaHome ":" $7}')
+groupEntry=$(getent group $wmaGroup)
+
+# workaround case where Unix account is not in the local system (e.g. sssd)
+[[ -d $HOST_MOUNT_DIR/etc/ ]] || (mkdir -p $HOST_MOUNT_DIR/etc) || exit $?
+if ! [ -f $HOST_MOUNT_DIR/etc/passwd ]; then
+    echo "Creating passwd file"
+    getent passwd > $HOST_MOUNT_DIR/etc/passwd
+    echo $passwdEntry >> $HOST_MOUNT_DIR/etc/passwd
+fi
+if ! [ -f $HOST_MOUNT_DIR/etc/group ]; then
+    echo "Creating group file"
+    getent group > $HOST_MOUNT_DIR/etc/group
+    echo $groupEntry >> $HOST_MOUNT_DIR/etc/group
+fi
+
+# create regular mount points at runtime
 [[ -d $HOST_MOUNT_DIR/certs ]] || (mkdir -p $HOST_MOUNT_DIR/certs) || exit $?
 [[ -d $HOST_MOUNT_DIR/admin/wmagent ]] || (mkdir -p $HOST_MOUNT_DIR/admin/wmagent) || exit $?
 [[ -d $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG/install ]] || (mkdir -p $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG/install) || exit $?
 [[ -d $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG/config  ]] || (mkdir -p $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG/config)  || exit $?
 [[ -d $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG/logs ]] || { mkdir -p $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG/logs ;} || exit $?
 
-chown -R $wmaUser $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG || exit $?
+# Check we own everything in $HOST_MOUNT_DIR/srv/wmagent/$WMA_TAG and $HOST_MOUNT_DIR/etc
```

Comment:
Same comment as above. What is the advantage of searching for all files not having the proper privileges, instead of recursively changing the ownership of the parent directory.  

Again this chown command here was doing nothing even before this change here, I myself was having in my mind the action item of removing it. And the reasons are  again the same: 
* if it is not `sudo` it will break and do nothing.
* if it is actually needed, it means only one thing someone did the mistake to run the container with the wrong user previously, so someone needs to go  there and clean the mess manually (only once, upon the mistake, and not on a regular basis)

Reply:
It looks like I have to remove this chown'ing from this file. Will do it later.